pages/juan-ortizc/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Carousel of image</title>
+  </head>
+  <body>
+    <section class="carousel">
+      <header>
+        <h1 class="sliders__tittle">
```

Comment:
You have a typo here. It should be a title

Reply:
I think this class name could be something like `carousel__title`, so that this uses the block name `carousel` and it's much clearer to know where this element belongs